src/base.ts

```diff
@@ -44,6 +44,16 @@ export abstract class TransactionCommand extends BlockchainCommand {
       default: "keystore",
       exclusive: ["address", "key"],
     }),
+    account: Flags.string({
+      helpGroup: "BASE",
+      description: "Account derivation number if using Ledger, starts at 0.",
+      helpValue: "ACCOUNT",
```

Comment:
s/ACCOUNT/N/

Reply:
@the-masthead are you asking me to change the `helpValue` from `ACCOUNT` to `s/ACCOUNT/N/`?

Reply:
Sorry, old slang. Asking to change ACCOUNT to N, for consistency with the rest of help strings.

Reply:
Oh duh it's sed hahaha. I was thinking that format looked familiar!